### ISSUE TITLE
Any chances on replacing tagflag to cobra

### ISSUE THREAD

**Issue by caryyu** [anacrolix/torrent#429]
Is there any plan to move tagflag to cobra https://github.com/spf13/cobra since it is very popular for  golang projects that sort of everyone knows well

**Comment by anacrolix** [comment]
Yes it's quite likely. tagflag is not keeping up, and documentation on cmd/torrent is falling behind.

**Comment by anacrolix** [comment]
@caryyu did you want to take a stab at doing this? I'm open to you applying your take on how hte flags should be done on any of the cmds in the repo, and I can extend it to the rest of them, or merge them all together into `cmd/torrent` if appropriate.

**Comment by anacrolix** [comment]
@caryyu what about https://github.com/alexflint/go-arg? Would that work?

**Comment by caryyu** [comment]
Hi @anacrolix, Sorry for a quite long time not replying to the above assignment, I don't think I fully understand the `tagflag` library right now, I will try to make an example to one of the `cmds` with above libraries, I think this assignment is quite formal, I'm afraid that I don't have much time to take this kind of responsibility, I'd better un-assign it, thanks.

**Comment by anacrolix** [comment]
No worries, thanks for the feedback. I've started on a clean-up of the cmds, currently with https://github.com/alexflint/go-arg. If there's any issues I'll try cobra.

**Comment by anacrolix** [comment]
This has been started with cmd/torrent, which now has a few subcommands:
```
Commands:
  download
  list-files
  spew-bencoding
  announce
```
So far so good. The issue remains open to contributions and suggestions.

**Comment by anacrolix** [comment]
I've started https://github.com/anacrolix/args, and it's used by more recent versions of anacrolix/torrent.
